tests/modules/vocab.txt

```diff
@@ -0,0 +1,19 @@
+a
+man
+with
+red
+helmet
+on
+small
+moped
+dirt
+road
+riding
+motor
+bike
+the
+countryside
+back
+of
+motorcycle
+<unk>
```

Comment:
Normally, <unk> is the first word in vocab.

Reply:
Actually upon checking we need not add the `<unk>` token as `BaseVocab` class adds this token. Removed this.